doc/api/console.md

```diff
@@ -7,7 +7,7 @@ JavaScript console mechanism provided by web browsers.
 
 The module exports two specific components:
 
-* A `Console` class with methods such as `console.log()`, `console.error()` and
+* A `ConsoleJefe` class with methods such as `console.log()`, `console.error()` and
```

Comment:
The rest of the changes in this file should also be removed, as this PR is supposed to be about renaming files only.